README.md

```diff
@@ -1,3 +1,92 @@
+-	Set markers for stimuli (cue markers) and output file (.txt file) of the times of the cue marks (this file will be used in the next step)
+-	I already have some of the CTF commands for this, but I would like to figure out how to give this script (and all of the following ones) a list of subject names to loop through so I can do this all at once.
+2.	Input: MEG data file
+3.	Output: copied and edited MEG data file, cue markers text file
+
+##	Behavior processing
+1.	Script name will be **02_behprocess**, and it will do the following:
+-	Makes marker files for more MEG processing; these markers will designate win/loss/neutral cues to be marked in the MEG file and are output as three separate .txt files to the subject’s MEG directory
+-	Pulls columns from behavior data file and calculates mean RTs and accuracy by trial and subject and appends that to master behavior data sheet (I will probably work with this sheet in R unless I can do an ANOVA as another step)
+-	I have already made two separate jupyter notebook files to do each of the above tasks, but I can figure how to do it without finding and replacing the subject ID; obviously a loop would be a lot easier, so I would like to figure that out and collapse both notebooks into one script
+2.	Input: cue markers text file, behavior text file
+3.	Output: win/loss/neutral text files (3), master file (csv? Not sure of best format for this) that includes mean reaction times and accuracy by participant and trial
+
+##	MEG preprocessing step 2
+1.	Script name will be **03_megpreprocess2**, and I would like it to do the following:
+-	Adds win/loss/neutral markers to cues
+-	Adds win/loss/neutral markers to responses
+-	Filter data based on existing filtering parameters file
+2.	Problems to figure out:
+-	When I do this one command at a time (which is how I’ve been doing this so far), it always misses two markers that I have to then add manually, so I need to figure out how to get it to add those two markers on its own.
+3.	Input: MEG file, win/loss/neutral marker files from the step above
+4.	Output: edited MEG file
+
+##	MEG processing with SAM
+1.	Script name will be **04_sam** and will do the following:
+-	orthohull (from subject’s MRI folder)
+-	Source localization in high gamma using three SAM commands and parameter file→ output is .nii file
+-	Copy .nii file from subject’s MEG directory to their MRI directory
+-	Talairach SAM results for analysis in afni
+-	Copy talairached .nii file into a group folder for group analysis
+2.	Problems:
+-	This step uses a parameter file that I would like to be able to create and distribute among subject directories whenever we want to change parameters (e.g. the time window we’re looking at in the task)
+-	Each of the three SAM commands takes a bit of time, but I heard that using swarm in biowulf may be a good way to do this. I am not sure how it would work with a loop just yet, but it’s an aspiration to make this somewhat efficient if possible.
```

Comment:
This is where nipype would shine. It is true that swarm would distribute this but then it is not very portable. And workflow managers do lots of intelligent management of the processing so that when you make a change you only recompute the bits of the analysis that you need to.